docs/pricing-and-billing.mdx

```diff
@@ -28,7 +28,7 @@ Semgrep's various functionalities are available through several offerings:
 <dt>Semgrep OSS</dt>
 <dd>The OSS (open source software) offering includes Semgrep OSS Engine, a fast static analysis command line tool for finding bugs and and enforcing code standards. It is licensed under LGPL 2.1. </dd>
 <dt>Semgrep Team tier</dt>
-<dd>The Team tier includes Semgrep Code and Semgrep Supply Chain (SSC), which enables users to scan both first-party and third-party open source dependencies. It also includes Semgrep Cloud Platform, a web app enabling users to manage users, organizations, repositories, and scans.</dd>
+<dd>The Team tier includes Semgrep Code and Semgrep Supply Chain (SSC), which enables users to scan both first-party and third-party open source dependencies. It also includes Semgrep Cloud Platform, a web app enabling users to manage users, organizations, repositories, and scans. Team tier also includes <strong>cross-function (intrafile)</strong> and <strong>cross-file (interfile) analysis</strong> through <a href="/semgrep-code/semgrep-pro-engine-intro/">Semgrep Pro Engine</a>.</dd>
```

Comment:
Can we turn the inclusions here into a bullet list? I _think_ that's allowed within a definition list, and it would be better now that there are three:

The Team tier includes:
* Semgrep Code and Semgrep Supply Chain (SSC), which enable users to scan both first-party and third-party open source dependencies. 
* <strong>Cross-function (intrafile)</strong> and <strong>cross-file (interfile) analysis</strong> through <a href="/semgrep-code/semgrep-pro-engine-intro/">Semgrep Pro Engine</a>.
* Semgrep Cloud Platform, a web app enabling users to manage users, organizations, repositories, and scans.

Reply:
Yep, resolved in latest :)